http/1.7/modules/ROOT/pages/http-polling-source-ref.adoc

```diff
@@ -0,0 +1,300 @@
+
+[source,xml,linenums]
+----
+<flow name="jsonPollingFlow">
+    <http:polling-source config-ref="requestConfig" path="json" splitExpression="#[payload]">
+        <scheduling-strategy>
+            <fixed-frequency frequency="10" timeUnit="SECONDS"/>
+        </scheduling-strategy>
+    </http:polling-source>
+    <logger message="#[payload]"/>
+</flow>
+----
+
+The following example shows two concurrent executions of the flow after one same polling, one execution with `<name>Mondi</name>` and another execution with `<name>Viena</name>`. After 10 hours, the polling is executed again. In this example, the server returns the same result:
+
+[source,xml,linenums]
+----
+<flow name="xmlListenerFlow">
+    <http:listener config-ref="listenerConfig" path="/xml"/>
+    <set-payload value="#[output application/xml --- {'something': {'element': [{name: 'Mondi'}, {name: 'Viena'}]}}]"/>
+</flow>
+
+<flow name="xmlPollingFlow">
+    <http:polling-source config-ref="requestConfig" path="xml"
+        splitExpression="#[payload.something.*element]">
+        <scheduling-strategy>
+            <fixed-frequency frequency="10" timeUnit="HOURS"/>
+        </scheduling-strategy>
+    </http:polling-source>
+    <logger message="#[payload]"/>
+</flow>
+----
+
+== Configure Watermarking Expression
+
+In the previous simple examples, the server always returned the same HTTP response. In more complex scenarios the server needs to know that it has to send the next response. You can send headers, URI parameters, body, or query parameters, but if these parameters always contain the same values in the HTTP request, the server does not know what would be the next response.
+
+For these polling scenarios, you use watermarking, where the server returns a watermark value on the payload itself, or in every item. For example, the watermark value can be a timestamp that can refer to the entire collection, or every item could have its own timestamp. In any case, you provide a watermark expression using the `watermark` placeholder.
+
+The expression extracts the watermark from the response, and subsequently, use this watermark to send the requests to the server. You can use expressions for the body, headers, URI parameters and query parameters' values in the the `watermark` placeholder.
+
+Note that in the first execution, the `watermark` placeholder value is `null`, which you might want to consider in the server or the expression where the placeholder is used.
+
+* To refer to the entire payload in the watermark expression, use the `payload` placeholder, as in the splitting expression.
```

Comment:
Consider reducing the use of texts that connect the event source to an HTTP request and the watermark expression to the splitting expression: I think that this assumes that the audience already understands one situation and can therefore understand the other, but that might not be the case. It seems risky to me.
```suggestion
* To refer to the entire payload in the watermark expression, use the `payload` placeholder.
```